test/Test/Database/LSMTree/Normal/StateMachine.hs

```diff
@@ -59,56 +74,133 @@ import qualified Test.QuickCheck.StateModel.Lockstep.Defaults as Lockstep.Defaul
 import qualified Test.QuickCheck.StateModel.Lockstep.Run as Lockstep.Run
 import           Test.Tasty (TestTree, testGroup)
 import           Test.Tasty.HUnit (testCase)
-import           Test.Util.Orphans (Wrap (..))
+import           Test.Tasty.QuickCheck (testProperty)
+import           Test.Util.Orphans ()
+import           Test.Util.TypeFamilyWrappers (WrapBlob (..), WrapBlobRef (..),
+                     WrapSession (..), WrapTableHandle (..))
 
 {-------------------------------------------------------------------------------
   Test tree
```

Comment:
INFO: there is quite a bit of duplication in this section between the `propLockstep*` functions, but I didn't want to define a general property yet (something for later, when these tests become more concrete)